server/src/router.ts

```diff
@@ -0,0 +1,17 @@
+import express from 'express';
+const router = express.Router();
+
+const controllers = require('./controllers/trip.controller.ts');
+
+//get all trips
+router.get('/trips', controllers.getAllTrips);
+
+//add trip
+router.post('/trips', controllers.addTrip);
+
+//added delete route to clean up database
+// router.delete('/trips/:id', controllers.deleteTrip);
```

Comment:
This delete route is useful, I think we should leave it in for now and not have it commented out

Reply:
done